package.json

```diff
@@ -34,6 +34,7 @@
     "imagemin": "^6.0.0",
     "jest": "^24.1.0",
     "mini-css-extract-plugin": "^0.7.0",
+    "null-loader": "^3.0.0",
```

Comment:
I'm concerned about the appearance of this being our primary recommended approach.  I think our main use case should be the simplest one where people use the react base css and embedded styles per component.  Otherwise they have to manually pull stylesheets or they get the whole thing.  We've gotten feedback that our component based loading has been well received and greatly simplified usage (as well as bundle size).  I'd much rather we include a different example somehow that shows advanced configuration options.

Reply:
I can move it to an example branch/wiki entry for those who might be interested. It really amounts to just a couple of changes, half of this PR is the lock file adjustments anyway :) - cool thx for the feedback!